### ISSUE TITLE
Discussion about merging upstream

### ISSUE THREAD

**Issue by luc-github** [Panucatt/RepRapFirmware#1]
Please check https://forum.duet3d.com/topic/13765/add-kinetica-g2-board-support

**Comment by docbobo** [comment]
Sorry, but the topic makes it sound as if nothing happened since January. Has this been merged into the mainline or not? Thanks.

**Comment by luc-github** [comment]
no, not yet, still same status, delayed due to several reasons - still under going

**Comment by luc-github** [comment]
@docbobo do you need something ? or have any request/question about code to be merged ?

**Comment by docbobo** [comment]
I believe there were two things I was trying to find out:

1. What repository would I have to go to for the latest source code? I believe this answer is "this".
2. Will there be any support for a 3.x firmware anytime soon? Based on the linked thread, I believe the answer is "no".

Based on the linked forum post, it's a little bit unclear to me what's preventing this from being merged into the other depot, but maybe I am just put too much weight on the idea of being able to always build from the original RepRap FW repo... ?

**Comment by luc-github** [comment]
1- yes
2 - currently no -  but should be next step depending Roy 

So for nothing prevent, the first step was to open discussion in forum,  the second was to do the missing iap porting in forum, and last one to do the PR on each repository
On same time the board was still under development and hardware tuning,  - so nothing was hurry
